Adjusted Loader widget documentation (#5230)
magento/devdocs: guides/v2.2/javascript-dev-guide/widgets/widget_loader.md

## Overview

> The Loader [widget](https://glossary.magento.com/widget) blocks page content (all content or a part of it). Its intended use is blocking content when an Ajax request is being sent. But it can be initialized for non-Ajax tasks as well. 

Reviewer: ```suggestion
The Loader [widget](https://glossary.magento.com/widget) blocks page content (all content or a part of it). Its intended use is blocking content when an Ajax request is being sent, but it can be initialized for non-Ajax tasks as well. 
```

The Loader widget source is [lib/web/mage/loader.js].

## Initialize the loader widget {#loader_init}

For information about how to initialize a widget in a JS component or `.phtml` template, see the [Initialize JavaScript][] topic.

> The Loader widget is instantiated with as in the following example:

Reviewer: ```suggestion
The following example shows how to instantiate the Loader widget:
```

## Initialize the loader widget {#loader_init}

For information about how to initialize a widget in a JS component or `.phtml` template, see the [Initialize JavaScript][] topic.

> The Loader widget is instantiated with the following:

Reviewer 2: Please, adjust this part.
```suggestion
The Loader widget is instantiated with as in the following example:
```

$("body").trigger('processStop');
```

> The following is an example of adding a callback to `processStop` event.

Reviewer: ```suggestion
The following is an example of adding a callback to a `processStop` event.
```

```javascript
var callback = function () {
    // do something before hiding the loader
};

$("body").on('processStop', callback);
```

## Code Sample

> The following example show loader when `processStart` event triggers for the `#loader-example` element.

Reviewer: ```suggestion
The following example shows the loader when a `processStart` event triggers for the `#loader-example` element.
```

```html
<div id="loader-example" data-mage-init='{"loader": { "icon": "{store URL}/static/{static version}/frontend/Magento/luma/en_US/images/loader-2.gif"}}'>
    <h3>Block</h3>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos itaque numquam placeat quam recusandae velit voluptas. Ad architecto asperiores eos eveniet id nostrum officiis saepe soluta totam voluptate! Debitis, quibusdam.
</div>
<script>
require(['jquery', 'loader', 'domReady!'], function ($) {
    'use strict';

    $('#loader-example').trigger('processStart');
});
</script>
```

## Result

> The result is loader shows when the `processStart` event triggers for `#loader-example` element.

Reviewer: ```suggestion
The loader shows when a `processStart` event triggers for the `#loader-example` element.
```
